efs-lib/src/cmr/efs/connection.clj

```diff
@@ -0,0 +1,39 @@
+(ns cmr.efs.connection
```

Comment:
When are the functions going to be written for EFS?  Is that another ticket that will be worked later?